features/settings/src/main/java/com/cryart/sabbathschool/settings/SSSettingsActivity.kt

```diff
@@ -51,7 +55,7 @@ class SSSettingsActivity : SSColorSchemeActivity() {
 
         supportFragmentManager.beginTransaction().replace(
             R.id.ss_settings_frame,
-            SSSettingsFragment()
+            SSSettingsFragment.newInstance(appConfig)
```

Comment:
You can inject appConfig inside `SSSettingsFragment` and not pass it as argument

Reply:
Unfortunately dagger cannot inject into a `PreferenceFragmentCompat`

Reply:
We can use a workaround described in [https://github.com/google/dagger/issues/2175](https://github.com/google/dagger/issues/2175). But, yeah, agree to leave it as is.